java/java.graph/src/org/netbeans/modules/java/graph/NodeWidget.java

```diff
@@ -102,25 +106,27 @@
 
     private String tooltipText;
     private final WidgetAction fixConflictAction;
-    private final WidgetAction sceneHoverActionAction; 
-    
+    private final WidgetAction sceneHoverActionAction;
+
     // for use from FruchtermanReingoldLayout
-    public double locX, locY, dispX, dispY; 
-    private boolean fixed; 
-    
-    NodeWidget(@NonNull DependencyGraphScene scene, GraphNode<I> node, final Action fixConflictAction, final WidgetAction sceneHoverActionAction) {        
+    public double locX, locY, dispX, dispY;
+    private boolean fixed;
+
+    NodeWidget(@NonNull DependencyGraphScene scene, GraphNode<I> node, final Action fixConflictAction, final WidgetAction sceneHoverActionAction) {
         super(scene);
         Parameters.notNull("node", node);
-        if(fixConflictAction != null) {
+        if (fixConflictAction != null) {
```

Comment:
Nit: Lots of whitespace changes in this commit makes it hard to review the real changes. It's a good idea to put whitespace changes in a separate commit (though no need to change it now).

Reply:
My Netbeans autoformatter has different setting than yours!